scripts/inc.compatibility.sh

```diff
@@ -263,15 +263,13 @@ webviewstock"
 
 api23hack(){
   if [ "$API" -ge "23" ]; then
-    gappscore="$gappscore
-packageinstallergoogle"
     gappspico="$gappspico
 googletts"
     gappsmini="$gappsmini
 calculatorgoogle"
     gappsstock="$gappsstock
```

Comment:
That's perfect for me, I just want it to be keyword configurable. Should I add it after googletts in L270 above?